doc/data/messages/i/import-private-name/bad.py

```diff
@@ -0,0 +1 @@
+from classes import _PrivateClassA, safe_get_A  # [import-private-name]
```

Comment:
We could use pytest there's a lot of import. We can also show that importing private import for typing purpose is ok.